### ISSUE TITLE
Can't get history message

### ISSUE THREAD

**Issue by duybuiappliedmesh** [webex/webex-android-sdk#57]
I used this function to get list message by room_id

**messageClient.list(roomId!!, before, 50, null) { result ->**

But I got an issue after back and request list few times

I got this error
OkHttp https://conv-r.wbx2.com/...  Parse json error: java.lang.ClassCastException: android.net.Uri$StringUri cannot be cast to java.lang.String
E/ServiceReqeust:263: OkHttp https://conv-r.wbx2.com/...  HTTP Error: WebexError[errorCode=UNEXPECTED_ERROR, message=Illegal body in the response, ]

Not sure why this happen, I don't touch anything, just go back and request few times, then issue will happen

**Comment by jeffmars** [comment]
This is an example of using messageClient.list 

```
        messageClient.list(roomId, null, 50, null, CompletionHandler->{
            if(CompletionHandler.isSuccessful()){
                Log.d(" Result 1","!!!!!!FETCHED MESSSAGES!!!!!!" );
            }
            else{
                Log.d(" Result 2","!!!!!!FAILED TO FETCH MESSAGES!!!!!!");
            }
        });
```

If you need further assistance troubleshooting your code please open a ticket with the developer support team by sending your details to devsupport@webex.com.

**Comment by duybuiappliedmesh** [comment]
> This is an example of using messageClient.list
> 
> ```
>         messageClient.list(roomId, null, 50, null, CompletionHandler->{
>             if(CompletionHandler.isSuccessful()){
>                 Log.d(" Result 1","!!!!!!FETCHED MESSSAGES!!!!!!" );
>             }
>             else{
>                 Log.d(" Result 2","!!!!!!FAILED TO FETCH MESSAGES!!!!!!");
>             }
>         });
> ```
> 
> If you need further assistance troubleshooting your code please open a ticket with the developer support team by sending your details to [devsupport@webex.com](mailto:devsupport@webex.com).

Yes, I already used this function and can get list message. BUT as my investigate, I found this issue above only happen when I join a call (mode auto record is open)
Let me sum my step
1. Join a room chat, send any message
2. Close app, then join room chat again to load list message (this step still OK)
3. Join a call with 5 minutes for example, then hang up. After some minutes (let say 15 - 20min)
4. Join room chat again => this issue happen

Does it related with mode auto record? I send you a log of 2 room I tested (one is join room chat only -> no issue, other one is join both chat and call -> issue)

[json_join_chat_only.txt](https://github.com/webex/webex-android-sdk/files/5905636/json_join_chat_only.txt)
[json_room_joined_call.txt](https://github.com/webex/webex-android-sdk/files/5905638/json_room_joined_call.txt)

**Comment by RahmanEckyRetnaldi** [comment]
excus me, how to make sending message and receive message to another participans on meeting room sdk.
